docs/course/course.yaml

```diff
@@ -121,4 +121,72 @@ paths:
               example:
                 status: 404
                 code: DOCUMENT_NOT_FOUND
-                message: Course with the specified id was not found.
\ No newline at end of file
+                message: Course with the specified id was not found.
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - Course
+      summary: Find one course.
+      description: Finds and returns one course.
+      produces:
+        - application/json
+      parameters:
```

Comment:
we don't need all these parameters of query besides `id`

Reply:
> we don't need all these parameters of query besides `id`

thank you for your advice, fixed that